modules/smtp/modules.php

```diff
@@ -1283,7 +1283,7 @@ protected function output() {
         if ($this->get('single_server_mode')) {
             return '';
         }
-        $count = count($this->get('smtp_servers', array()));
+        $count = count(array_filter($this->get('smtp_servers', array()), fn($s) => ($s['type'] ?? null) !== 'ews'));
```

Comment:
SMTP servers cannot have ews type. EWS servers are added to the imap list and distinguished there by the type.